### ISSUE TITLE
Fail if no version is specified in `pack build` when multiple versions exist in builder

### ISSUE THREAD

**Issue by ssisil** [buildpack/pack#160]
**Scenario : Buildpack version is not set as latest in builder.toml**
__GIVEN__ I have a CNB with an ID of Z and  a version set to X
__AND__ I have a CNB with an ID of Z and a version set to Y
__AND__ neither are set as the latest in the [[buildpacks]] section of my builder.toml 
__WHEN__ I run `pack build <image name> --builder <my builder> --buildpack Z
__THEN__ the build should fail
__AND__ the following message should be displayed to the developer
```
<image name> failed to build.  <my builder> has multiple versions of buildpack Z, please provide the specific version you want to use. 
```

**Comment by ekcasey** [comment]
We added a validation similar to this when we refactored the build command. I want to call this done, but would be happy to incorporate any feedback about the specific text of the error message.

**Comment by mgibson1121** [comment]
I'm assuming the error you wrote @ekcasey is `ERROR: failed to set custom buildpack order: there is no version of buildpack org.cloudfoundry.buildpacks.nodejs marked as latest`?  Based on the story title, the scenario here is that a build fails because you specify an out of date version of a bp in a builder, and then reference that buildpack by its ID (which is set to latest), causing a car accident.  

I think it's good enough for now, accepting

**Comment by ekcasey** [comment]
fixed on `master`, will ship in `v0.2.0`
